Functions/Get-ShouldOperator.ps1

```diff
@@ -65,12 +65,14 @@ function Get-ShouldOperator {
 
     END {
         if ($Name) {
+           
```

Comment:
This file contains just unnecesary whitespace changes, please revert.

Reply:
This has been removed in last commit